TESTING.md

```
@@ -0,0 +1,621 @@
+# Testing Strategy for MoveHat
+
+Complete testing and validation guide before publishing.
+
+##  Table of Contents
```

markdownlint-cli2 (0.22.1)

[warning] Multiple spaces after hash on atx style heading

(MD019, no-multiple-space-atx)